.github/workflows/wasm_demos.yaml

```diff
@@ -82,6 +82,7 @@ jobs:
                       demos/energy-monitor/
                       demos/home-automation/rust
                       demos/weather-demo/
+                      demos/usecases/rust
```

Comment:
This particular step is not run in the regular CI, but if it were this would fail, as the demo isn't built. So merging this PR will break the nightly build. I suggest to either remove this and fix it afterwards, or fix it in this PR by including the demo in our WASM demos.


Generally, it would be nice to have *some* CI coverage for the slint files.

Reply:
> Generally, it would be nice to have some CI coverage for the slint files.

Could be added in https://github.com/slint-ui/slint/blob/22317d29b4f6dbe1a3e30ab4c1fc64398cf0b900/tests/driver/interpreter/main.rs#L33